oteapi/strategies/parse/image.py

```diff
@@ -22,19 +22,36 @@ class ImageParserConfig(AttrDict):
 
     crop: Optional[Tuple[int, int, int, int]] = Field(
         None,
-        description="Box cropping parameters.",
+        description="Box cropping parameters (left, top, right, bottom).",
     )
     datacache_config: Optional[DataCacheConfig] = Field(
         None,
         description="Configuration options for the local data cache.",
     )
+    download_config: Optional[ResourceConfig] = Field(
+        {},
```

Comment:
The default value should be either `None` or `ResourceConfig()` - see also the given type for this attribute (`Optional[X]` = `Union[None, X]`).

Reply:
done